lib/qa/authorities/users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# @abstract This duplicates Qa::Authorities::Persons with the only difference being only Penn State users are returned
+# in a search. An alternative implementation would be to define this as a subauthority of Persons, but this just seems
+# easier.
+module Qa
+  module Authorities
+    class Users < Persons
+      private
+
+        # @return Array<PennState::SearchService::Person>
+        def persons
+          identities
```

Comment:
This is some kinda magical mystery code, but everything looks good to me.

Reply:
@rschenk it's in the super class. Persons is a combination of local actors and Penn State people (which is the `identities` method). But here, we just want the Penn Staters... so, identities. Not very intuitive.